_minutes/2024-02-15-wecg.md

```diff
@@ -0,0 +1,158 @@
+   * [Issue 523](https://github.com/w3c/webextensions/issues/523): API limitation: content scripts URL matches and History API, Navigation API, BFCache
+   * [Issue 527](https://github.com/w3c/webextensions/issues/527): chrome.scripting.executeScript doesn't resolve/reject for frozen state tabs
+ * **Timely issues - 10 minutes** _(Hard cut-off at 40 minutes past)_
+   * [Issue 540](https://github.com/w3c/webextensions/pull/540): Add userScripts.execute() API proposal
+   * [Issue 433](https://github.com/w3c/webextensions/issues/433): Inconsistency: Alarms API time slipping
+     * Chrome 123 uses wall clock time
+     * Should we keep the issue open for fixing Firefox sleep behavior and keeping a consistent schedule for repeating alarms?
+   * [Issue 401](https://github.com/w3c/webextensions/issues/401): ​​Expose SHA256 value of DownloadItem
+   * [Issue 454](https://github.com/w3c/webextensions/issues/454): Add a new manifest key to include a origin trial token
+ * **Check-in on existing issues** **- 20 minutes**
+   * [Issue 519](https://github.com/w3c/webextensions/issues/519): Proposal: Event.addListener should accept an interface.
+   * [Follow up](https://github.com/w3c/webextensions/issues/402#issuecomment-1591619962) on [Issue 402](https://github.com/w3c/webextensions/issues/402): Provide a means to discriminate initiators and destinations belonging to a local (private) network
+
+
+## Attendees (sign yourself in)
+
+ 1. Rob Wu (Mozilla)
+ 2. Patrick Kettner (Google)
+ 3. Simeon Vincent (unaffiliated)
+ 4. Philippe Le Hegaret (W3C)
+ 5. Giorgio o Maone (Tor, NoScript)
+ 6. Richard Worth (Capital One)
+ 7. Carlos Jeurissen (Jeurissen Apps)
+ 8. Tim Heflin (Keeper)
+ 9. David Johnson (Apple)
+ 10. Dmitriy Seregin (AdGuard)
+ 11. Anton Bershanskyi
+ 12. Emilia Paz (Google)
+ 13. Maxim Topciu (AdGuard)
+ 14. Todd Schiller (PixieBrix)
+ 15. Mukul Purohit (Microsoft)
+ 16. Dávid Tóta (AdGuard)
+
+
+## Meeting notes
+
+Before start of meeting:
+
+ * [plh] I'm from the W3C. We're curious about why this is a CG and not a WG?
+ * [simeon] Answering that will take more time than we have on the agenda. I'll invite you to a separate meeting to discuss that.
+ * [simeon] Reminder: this is the first meeting with the new agenda format sketched in [issue 531](https://github.com/w3c/webextensions/issues/531). Thanks Oliver for preparing the template and agenda for today!
+
+[Issue 522](https://github.com/w3c/webextensions/issues/522): Proposal: Give extensions easy access to MediaSession
+
+ * [simeon] Request is to expose controls to MediaSession so that extensions don't need all_urls permission to interact with playback of a tab.
+ * [oliver] Interesting capability. For prioritization, having use cases would be good.
+ * [rob] Came from a bugzilla issue and we asked them to open a ticket. Niche feature, but sounds useful. Exposing MediaSession could fit within the existing API. Ideal next step is for someone to prepare a proposal and prepare a patch.
```

Comment:
Did someone volunteer? It seems like that be covered by the minutes either way.

Reply:
Nobody volunteered during the meeting; I posted a comment with the status at https://github.com/w3c/webextensions/issues/522#issuecomment-1946660129